virtcontainers/netmon/netmon.go

```diff
@@ -0,0 +1,530 @@
+// Copyright (c) 2018 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/kata-containers/agent/protocols/grpc"
+	"github.com/vishvananda/netlink"
+	"golang.org/x/sys/unix"
+)
+
+const (
+	netmonName    = "kata-netmon"
+	netmonVersion = "0.0.1"
+
+	kataCLIAddIfaceCmd   = "add-interface"
+	kataCLIDelIfaceCmd   = "del-interface"
+	kataCLIUpdtRoutesCmd = "update-routes"
+
+	kataSuffix = "kata"
+
+	netlinkFamily = netlink.FAMILY_V4
+
+	storageParentPath = "/var/run/kata-containers/netmon/sbs"
```

Comment:
this path should be generated by in the Makefile

Reply:
Good point.